src/test/java/pageobjects/SalesforceObjects/CampaignObject.java

```diff
@@ -74,11 +107,14 @@ public static By clickParentLink(String campaignName){
     public static By showActions(String campaignName){
         return By.xpath("//table[@role=\"grid\"]//tr//th[@scope=\"row\"]//a[text()='"+campaignName+"']//ancestor::th//following-sibling::td[8]//a//span[text()='Show Actions']");
     }
-    public static WebElement select_Tab(String tabname){
```

Comment:
is method select_tab not required anymore?

Reply:
no it is not required